plugins/feeds/public/hostsfile_emd.py

```diff
@@ -0,0 +1,52 @@
+from datetime import datetime, timedelta
+import logging
+
+from core.observables import Hostname
+from core.feed import Feed
+from core.errors import ObservableValidationError
+
+class HostsFile_EMD(Feed):
+	default_values = {
+		'frequency': timedelta(hours=4),
+		'source': 'https://hosts-file.net/emd.txt',
+		'name': 'Hosts_File_EMD',
+		'description': 'Sites engaged in malware distribution.'
+
+	}
+
+	def set(self, type, value):
```

Comment:
Obviously, I just failed to cleanup appropriately :)... I need to go back through and make sure the repo matches what I was doing in testing.